staging/src/k8s.io/pod-security-admission/admission/admission.go

```diff
@@ -407,7 +407,10 @@ func (a *Admission) EvaluatePod(ctx context.Context, nsPolicy api.Policy, nsPoli
 		klog.V(2).InfoS("failed to parse PodSecurity namespace labels", "err", nsPolicyErr)
 		auditAnnotations["error"] = fmt.Sprintf("Failed to parse policy: %v", nsPolicyErr)
 	}
-	// TODO: log nsPolicy evaluation with context (op, resource, namespace, name) for the request.
+
+	if klog.V(4).Enabled() {
```

Comment:
From the discussions on slack (https://kubernetes.slack.com/archives/C0EG7JC6T/p1632866835339300):

> liggitt: per-request http logs are at V3 (httplog.go#WithLogging), so definitely higher than that. PodSecurityPolicy admission logged at V4, though this will be on by default on more resources... I'd waffle between V4 and V5
> dims: +1 to V5

Let's bump this up to `V(5)`